README_NEW.md

```diff
@@ -0,0 +1,654 @@
+make operator_deposit_into_mock_strategy
+```
+
+#### Holesky/Mainnet
+
+EigenLayer strategies are available in [eigenlayer-strategies](https://holesky.eigenlayer.xyz/restake).
+
+For Holesky, we are using [WETH](https://holesky.eigenlayer.xyz/restake/WETH) as the strategy token.
+
+To obtain HolETH and swap it for different strategies, you can use the following [guide](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/stage-2-testnet/obtaining-testnet-eth-and-liquid-staking-tokens-lsts).
+
+### Register into AlignedLayer
+
+To register an operator in AlignedLayer run the following command:
+
+```bash
+make operator_register_with_aligned_layer CONFIG_FILE=<path_to_config_file>
+```
+
+To register an operator in AlignedLayer with the default configuration, you can run:
+
+```bash
+make operator_register_with_aligned_layer
+```
+
+### Full Registration in Anvil
+
+For devnet purposes, you can run the following command to register an operator in EigenLayer and AlignedLayer and deposit strategy tokens in EigenLayer:
+
+```bash
+make operator_full_registration CONFIG_FILE=<path_to_config_file>
+```
+
+To register an operator in EigenLayer and AlignedLayer and deposit strategy tokens in EigenLayer with the default configuration, you can run:
+
+```bash
+make operator_full_registration
+```
+
+### Run
+
+To start the operator run:
+
+```bash
+make operator_start CONFIG_FILE=<path_to_config_file>
+```
+
+If you want to run the operator with the default configuration, you can run:
+
+```bash
+make operator_start
+```
+
+### Config
+
+There is a default configuration for devnet purposes in `config-files/config.yaml`.
+Also, there are 3 different configurations for the operator in `config-files/devnet/operator-1.yaml`, `config-files/devnet/operator-2.yaml` and `config-files/devnet/operator-3.yaml`.
+
+The configuration file has the following structure:
+
+```yaml
+# Common variables for all the services
+# 'production' only prints info and above. 'development' also prints debug
+environment: <production/development>
+aligned_layer_deployment_config_file_path: <path_to_aligned_layer_deployment_config_file>
+eigen_layer_deployment_config_file_path: <path_to_eigen_layer_deployment_config_file>
+eth_rpc_url: <http_rpc_url>
+eth_ws_url: <ws_rpc_url>
+eigen_metrics_ip_port_address: <ip:port>
+
+## ECDSA Configurations
+ecdsa:
+  private_key_store_path: <path_to_ecdsa_private_key_store>
+  private_key_store_password: <ecdsa_private_key_store_password>
+
+## BLS Configurations
+bls:
+  private_key_store_path: <path_to_bls_private_key_store>
+  private_key_store_password: <bls_private_key_store_password>
+
+## EigenDA Configurations
+eigen_da_disperser:
+  url: <eigen_da_disperser_url> # This is the url of the EigenDA Disperser
+
+## Celestia Configurations
+celestia:
+  url: <celestia_url> # This is the url of the deployed Celestia Light node
+  keystore: <celestia_keystore> # This is the keystore of the Celestia Light node
```

Comment:
Remove this

Reply:
Solved in [c8752dd](https://github.com/yetanotherco/aligned_layer/pull/152/commits/c8752ddbefedbcb4a80d4847468e26047b780a65)